src/pages/health-declaration/__tests__/health-declaration.spec.ts

```diff
@@ -235,13 +235,12 @@ describe('HealthDeclarationPage', () => {
         const declarationIntent = fixture.debugElement.query(By.css('h4')).nativeElement;
         expect(declarationIntent.innerHTML).toBe('I declare that:');
       });
-      it('should render the page in Welsh for a Welsh test', (done) => {
+      it('should render the page in Welsh for a Welsh test', () => {
         fixture.detectChanges();
         translate.onLangChange.subscribe(() => {
           fixture.detectChanges();
           const declarationIntent = fixture.debugElement.query(By.css('h4')).nativeElement;
           expect(declarationIntent.innerHTML).toBe('[CY] I declare that:');
-          done();
```

Comment:
The assertions inside this subscribe won't fire without it being an async test with `done`, will they?

Reply:
could try a console.log after the assertion to see if it reaches that far in the test

Reply:
Unless it is throwing a false positive I believe the assertion fires. If I reinstate the `done`, the async times out.

Reply:
Confirmed, false positive!